R/zchunk_L202.an_input.R

```diff
@@ -0,0 +1,646 @@
+      add_title("Base year output of the inputs (feed types) to animal production") %>%
+      add_units("units") %>%   # TODO - what is calOutputValue units here?
+      add_comments("Animal production input-output coefficients written to all regions and base years") %>%
+      add_legacy_name("L202.StubTechProd_in") %>%
+      add_precursors("aglu/A_an_input_technology", "L107.an_FeedIO_R_C_Sys_Fd_Y",
+                     "energy/A_regions", "common/GCAM_region_names") ->
+      L202.StubTechProd_in
+
+    L202.Supplysector_an %>%
+      add_title("Generic animal production supplysector info") %>%
+      add_units("NA") %>%
+      add_comments("A_an_supplysector written to all regions") %>%
+      add_legacy_name("L202.Supplysector_an") %>%
+      add_precursors("aglu/A_an_supplysector", "common/GCAM_region_names") ->
+      L202.Supplysector_an
+
+    L202.SubsectorAll_an %>%
+      add_title("Generic animal production subsector info") %>%
+      add_units("NA") %>%
+      add_comments("A_an_subsector written to all regions") %>%
+      add_legacy_name("L202.SubsectorAll_an") %>%
+      add_precursors("aglu/A_an_subsector", "common/GCAM_region_names") ->
+      L202.SubsectorAll_an
+
+    L202.StubTech_an %>%
+      add_title("Identification of stub technologies for animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_technology written to all regions") %>%
+      add_legacy_name("L202.StubTech_an") %>%
+      add_precursors("aglu/A_an_technology", "common/GCAM_region_names") ->
+      L202.StubTech_an
+
+    L202.StubTechInterp_an %>%
+      add_title("Shareweight interpolation for animal production technologies") %>%
+      add_units("NA") %>%
+      add_comments("A_an_technology written to all regions") %>%
+      add_legacy_name("L202.StubTechInterp_an") %>%
+      add_precursors("aglu/A_an_technology", "common/GCAM_region_names") ->
+      L202.StubTechInterp_an
+
+    L202.StubTechProd_an %>%
+      add_title("Animal production by technology and region") %>%
+      add_units("Unitless") %>%
+      add_comments("Animal production written across model base years and regions") %>%
+      add_legacy_name("L202.StubTechProd_an") %>%
+      add_precursors("aglu/A_an_technology", "L107.an_Prod_Mt_R_C_Sys_Fd_Y", "common/GCAM_region_names") ->
+      L202.StubTechProd_an
+
+    L202.StubTechCoef_an %>%
+      add_title("Animal production input-output coefficients by technology and region") %>%
+      add_units("Unitless") %>%
+      add_comments("Animal production input-output coefficients written across model base years and regions") %>%
+      add_legacy_name("L202.StubTechCoef_an") %>%
+      add_precursors("aglu/A_an_technology", "L107.an_Prod_Mt_R_C_Sys_Fd_Y", "common/GCAM_region_names") ->
+      L202.StubTechCoef_an
+
+    L202.GlobalTechCost_an %>%
+      add_title("Costs of animal production technologies") %>%
+      add_units("billion USD") %>%
+      add_comments("Animal feed cost, prices, and technology") %>%
```

Comment:
Units: 1975$/kg
Comments: This is the non-feed cost; i.e., all costs of producing animal commodities except for the feed.